cmd/kas-grants.go

```diff
@@ -153,6 +155,80 @@ func policy_unassignKasGrant(cmd *cobra.Command, args []string) {
 	HandleSuccess(cmd, "", t, res)
 }
 
+func policy_listKasGrants(cmd *cobra.Command, args []string) {
+	c := cli.New(cmd, args)
+	h := NewHandler(c)
+	defer h.Close()
+	kasF := c.Flags.GetOptionalString("kas")
+	var (
+		kasID  string
+		kasURI string
+	)
+
+	// if not a UUID, infer flag value passed was a URI
+	if kasF != "" {
+		_, err := uuid.Parse(kasF)
+		if err != nil {
+			kasURI = kasF
+		} else {
+			kasID = kasF
+		}
+	}
+
+	grants, err := h.ListKasGrants(cmd.Context(), kasID, kasURI)
```

Comment:
Does this return all kas grants in the db if `kasF` is an empty string? Just verifying that sure the `--kas` flag is optional.

Reply:
Yes, precisely. You can see that demonstrated here in the e2e tests added for this functionality: https://github.com/opentdf/otdfctl/pull/346/files#diff-53724192e0c3172f28c28158e89035dd43c5358ca19a4d322a3072050af48c43R49-R66